src/ru/skypro/Main.java

```diff
@@ -3,6 +3,92 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+
+        //Задание 1
+
+        int clientOS = 1;
+        if (clientOS == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+        if (clientOS == 0) {
+            System.out.println("Установите версию приложения для IOS по ссылке");
+        }
+
+
+        // Задание 2
+
+        int clientDeviceYear = 2015;
+        int clientOs = 0;
+        if (clientDeviceYear < 2015 && clientOs == 1)
+            System.out.println("Установите облегченную версию приложения для Android по ссылке");
+        if (clientDeviceYear >= 2015 && clientOs == 1) {
+            System.out.println("Установите обычную версию приложения для Android по ссылке");
+        } else if (clientDeviceYear < 2015 && clientOs == 0) {
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        } else if (clientDeviceYear >= 2015 && clientOs == 0) {
+            System.out.println("Установите обычную версию приложения для iOS по ссылке");
+        }
```

Comment:
лучше объединить все условия в один оператор
```suggestion
        if (clientDeviceYear < 2015 && clientOs == 1) {
            System.out.println("Установите облегченную версию приложения для Android по ссылке");
        } else if (clientDeviceYear >= 2015 && clientOs == 1) {
            System.out.println("Установите обычную версию приложения для Android по ссылке");
        } else if (clientDeviceYear < 2015 && clientOs == 0) {
            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
        } else if (clientDeviceYear >= 2015 && clientOs == 0) {
            System.out.println("Установите обычную версию приложения для iOS по ссылке");
        }
```